### DISCUSSION TITLE
Exposing server across devices

### DISCUSSION THREAD

**Discussion by t-rkrosnick** [microsoft/FluidFramework#6482] [Q&A]
I'm trying to use Fluid Framework for clients that are on different devices, and trying to test it on my local network, e.g., with my phone and my laptop. Do you have any suggestions for local testing like this? For example, I'm building on the tutorial (Tutorial (fluidframework.com)) that uses tinylicious which runs on localhost:3000 -- can I adjust this to explicitly hardcode my current IP address? Or if you have any other suggestions on best practices for local testing across devices, that would be welcome.

**Comment by tylerbutler** [comment]
@t-rkrosnick Hello! We have some new documentation about multi-client testing with Tinylicious on fluidframework.com. See <https://fluidframework.com/docs/testing/tinylicious/#testing-with-tinylicious-and-multiple-clients> for more. Please feel free to post back here if you have questions or issues.
